tests/test_docs_admonition_inserter.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python
+#
+# A library that provides a Python interface to the Telegram Bot API
+# Copyright (C) 2015-2023
+# Leandro Toledo de Souza <devs@python-telegram-bot.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+
+import pytest
+
+import telegram.ext
+from docs.auxil.admonition_inserter import AdmonitionInserter
+
+
+@pytest.fixture(scope="session")
+def admonition_inserter():
+    return AdmonitionInserter()
+
+
+class TestAdmonitionInserter:
```

Comment:
```suggestion
class TestAdmonitionInserter:
    """This is a minimal effort test to ensure that the `AdmonitionInserter` used for automatically inserting references in the docs works as expected. It does not aim to cover all links in the documentation, but rather checks that several special cases are handled correctly which where discovered during the implementation of `AdmonitionInserter`
    """
```

Feel free to rephrase to your liking - a general note in that direction will be useful for future-us though :)